src/index.js

```diff
@@ -16,23 +18,37 @@ const todos = [{
   index: 0,
 },
 ];
+if (!window.localStorage.getItem('todos')) {
+  window.localStorage.setItem('todos', JSON.stringify(dummyTodos));
+  todos = dummyTodos;
+} else {
+  todos = JSON.parse(window.localStorage.getItem('todos'));
+}
+
 const arrSort = (arr) => arr.sort((a, b) => a.index - b.index);
 
-// function component() {
 const list = document.querySelector('ul');
 const sortedTodos = arrSort(todos);
 if (sortedTodos.length > 0) {
   sortedTodos.forEach((el) => {
     const todo = document.createElement('div');
-    todo.setAttribute('class', 'todoItem');
+    todo.setAttribute('id', `${el.index}`);
+    todo.classList.add('todoItem');
     const titleCheckbox = document.createElement('div');
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.addEventListener('click', (e) => {
+      const id = e.target.parentNode.parentNode.getAttribute('id');
```

Comment:
- [ ] Kindly ensure that when I click to complete a to-do the completed state for that particular to-do actually gets saved in the localstorage. So click a to do item and refresh the page and the to do checkbox should still be checked for that item.

Reply:
- Hi, could you make it so when the user clicks the checkbox and refreshes the page the checkbox **still remains checked**.

Reply:
Hi. Thanks for clarifying. Now I see what you wanted. I've fixed it so that the checkbox remains checked for completed items.

Reply:
This issue is well resolved! ✔️